worlds/cvcotm/items.py

```diff
@@ -0,0 +1,203 @@
+    # Dash Boots
+    iname.double:         CVCotMItemData(0xE801, b"\x59\x81", ItemClassification.progression, b"\xF4\x84"),
+    iname.tackle:         CVCotMItemData(0xE802, b"\x5A\x81", ItemClassification.progression, b"\xF5\x84"),
+    iname.kick_boots:     CVCotMItemData(0xE803, b"\x5B\x81", ItemClassification.progression, b"\xF6\x84"),
+    iname.heavy_ring:     CVCotMItemData(0xE804, b"\x5C\x81", ItemClassification.progression, b"\xF7\x84"),
+    # Map
+    iname.cleansing:      CVCotMItemData(0xE806, b"\x5D\x81", ItemClassification.progression, b"\xF8\x84"),
+    iname.roc_wing:       CVCotMItemData(0xE807, b"\x5E\x81", ItemClassification.progression, b"\xF9\x84"),
+    iname.last_key:       CVCotMItemData(0xE808, b"\x5F\x81", ItemClassification.progression_skip_balancing,
+                                         b"\xFA\x84"),
+    iname.ironmaidens:    CVCotMItemData(0xE809, b"\xF1\x84", ItemClassification.progression),
+    iname.dracula:        CVCotMItemData(None, None, ItemClassification.progression),
+    iname.shinning_armor: CVCotMItemData(None, None, ItemClassification.progression),
+}
+
+ACTION_CARDS = {iname.mercury, iname.venus, iname.jupiter, iname.mars, iname.diana, iname.apollo, iname.neptune,
+                iname.saturn, iname.uranus, iname.pluto}
+
+ATTRIBUTE_CARDS = {iname.salamander, iname.serpent, iname.mandragora, iname.golem, iname.cockatrice, iname.griffin,
+                   iname.manticore, iname.thunderbird, iname.unicorn, iname.black_dog}
+
+FREEZE_ACTIONS = [iname.mercury, iname.mars]
+FREEZE_ATTRS = [iname.serpent, iname.cockatrice]
+
+FILLER_ITEM_NAMES = [iname.heart_max, iname.hp_max, iname.mp_max]
+
+MAJORS_CLASSIFICATIONS = ItemClassification.progression | ItemClassification.useful
+
+
+def get_item_names_to_ids() -> Dict[str, int]:
+    return {name: cvcotm_item_info[name].code + BASE_ID for name in cvcotm_item_info
+            if cvcotm_item_info[name].code is not None}
+
+
+def get_item_counts(world: "CVCotMWorld") -> Dict[ItemClassification, Dict[str, int]]:
+
+    item_counts: Dict[ItemClassification, Dict[str, int]] = {
+        ItemClassification.progression: {},
+        ItemClassification.progression_skip_balancing: {},
+        ItemClassification.useful: {},
+        ItemClassification.filler: {},
+    }
+    total_items = 0
+    # Items to be skipped over in the main Item creation loop.
+    excluded_items = [iname.hp_max, iname.mp_max, iname.heart_max, iname.last_key]
+
+    # If Halve DSS Cards Placed is on, determine which cards we will exclude here.
+    if world.options.halve_dss_cards_placed:
+        excluded_cards = list(ACTION_CARDS.union(ATTRIBUTE_CARDS))
+
+        has_freeze_action = False
+        has_freeze_attr = False
+        start_card_cap = 8
+
+        # Get out all cards from start_inventory_from_pool that the player isn't starting with 0 of.
+        start_cards = [item for item in world.options.start_inventory_from_pool.value if "Card" in item and
```

Comment:
There are no benefits, yeah. Like I replied in a different comment, having duplicates of the same card is pointless because you only ever need one instance to use all of the combos associated with it, and you don't ever lose cards.